### PR TITLE
Added MeterAria

### PR DESCRIPTION
**Pull request by bolu-tife**
# What Changed
Added ARIA meter role in react-snippets.code-snippets 

# Why
Solution to #58  

Todo:
- [ ]  Add Semantic Version Label 
- [x] Add tests/Tested on local machine
- [x] Add docs

### CODE REVIEW

File: snippets/react-snippets.code-snippets [hunk 1]
```diff
@@ -72,6 +72,23 @@
...
+            "aria-label='$1'",
+            "aria-labelledby='$2'",
+            "aria-describedby='$3'",
+            "aria-valuemax={$4}",
+            "aria-valuemin={$5}",
+            "aria-valuenow={$6}",
+            "aria-valuetext='$7'",
+            ""
+        ],
+        "description": "Add `role='meter'` to identify an element being used as a meter. A meter is a graphical display of a numeric value within a defined range. Each element must have either an `aria-label` or `aria-labelledby`. Add `aria-labelledby` to point to an element with text that describes the meter. Add `aria-valuenow` to indicate the current value of the meter, which is between the `aria-valuemin` and `aria-valuemax`. Add `aria-valuetext` to make the meter value understandable. Add `aria-valuemin` to indicate the minimum value, which is less than `aria-valuemax`. Add `aria-valuemax` to indicate the maximum value, which is greater than `aria-valuemin`. "
```
Comment (kendallgassner) [inline snippets/react-snippets.code-snippets:90]: would you mind adding this description to the readme too!

Comment (bolu-tife) [reply snippets/react-snippets.code-snippets]: I'm on it already😃

**Comment by bolu-tife** [comment]
# What Changed
Updated README.md with ARIA meter role 

# Why
Solution to #62 

Todo:
- [ ]  Add Semantic Version Label 
- [ ] Add tests/Tested on local machine
- [x] Add docs

**Comment by kendallgassner** [comment]
Minor comment! We have updated the readme to be formatted as a table! If you can update you pr + edit in the readme to align with the change that would be amazing 🙏 thank you!!

**Comment by bolu-tife** [comment]
I have updated the README.md to accommodate the new format in #64.

**Comment by kendallgassner** [comment]
I am not seeing the meter aria in the readme .. It looks like you might have forgotten to add it

**Pull request merged**